### DISCUSSION TITLE
TEID -2147483643 not found on GTP-U Downlink

### DISCUSSION THREAD

**Discussion by rajyay** [magma/magma#15446] [General]
Hi Community, Thanks for your help so far. Now, i am facing the error as seen in the attached image while ping 8.8.8.8 -I uesimtun0 does not work after PDU session establishment and tunnel interface creation. Kindly request you to help me.
![image](https://github.com/magma/magma/assets/2726718/5d90436f-d18d-4907-8e13-16e4aba8b2e9). i have gone through https://github.com/magma/magma/issues/14623 but could not solve my issue

**Comment by lucasgonze** [comment]
Is this still an unsolved problem? If not could you document your solution?

**Reply by rajyay** [reply]
Hi, this is solved now by commenting out the timestamp field in file UERANSIM/src/gnb/ngap/transport.cpp function NgapTask::sendNgapUeAssociated as discussed in https://github.com/magma/magma/issues/12846

**Reply by lucasgonze** [reply]
Thank you for posting this workaround. I believe the fix will be incorporated in the 1.9 release.

**Reply by rajyay** [reply]
Hi, sorry i got confused but this issue is not solved till now. What i solved was problem of PDU establishment. But, after PDU session is esatablished, still I am not being able to solve the "TEID -2147483643 not found on GTP-U Downlink" issue
